libs/packages/components/src/lib/video-player/video-player.ts

```diff
@@ -10,4 +10,5 @@ export interface VPInterface{
  	debug: boolean;
   preload: string;
   description?: string;
+  crossorigin?: string;
```

Comment:
We already have a crossorigin input as a separate input (not part of the model) and it's added dynamically.